### ISSUE TITLE
Shogun is not up to date with Shogun-cpp

### ISSUE THREAD

**Issue by CJ-Wright** [conda-forge/shogun-feedstock#24]
The shogun-cpp pin is aligned to the version of shogun but those two versions have diverged leading to solvability errors for the arpack37 migration.

**Comment by djsutherland** [comment]
Yeah, sorry about that; we're blocked on conda-forge/shogun-feedstock#23 right now.

Since I don't think anything in conda-forge depends on `shogun` (though I haven't checked), you could just mark it as migrated maybe; I put the migration in #23 to use the latest shogun builds.

**Comment by CJ-Wright** [comment]
I can wait, I just don't know if you knew it was an issue. The bot doesn't expose it's solvability errors well.
